src/app/chat/event_handling.nim

```diff
@@ -77,11 +77,9 @@ proc handleChatEvents(self: ChatController) =
 
   self.status.events.on(PendingTransactionType.BuyStickerPack.confirmed) do(e: Args):
     var tx = TransactionMinedArgs(e)
+    self.view.transactionCompleted(tx.success, tx.transactionHash)
```

Comment:
No big deal, but we can pass the `revertReason` already even if we don't listen to it in QML yet